scenario/README

```diff
@@ -0,0 +1,24 @@
+By default CloudFerry tool is going to migrate all resources and instances from the source cloud to the destination cloud.
```

Comment:
The doc looks great, please modify it to use [markdown](http://daringfireball.net/projects/markdown/syntax).